modules/github/standardized-repository-settings/main.tf

```diff
@@ -0,0 +1,124 @@
+/**
+  * This module sets up a GitHub repository in a standardized way, to ensure
+  * consistency among things like branch and PR handling, labels and so on.
+  */
+
+terraform {
+  required_version = "0.15.3"
+
+  required_providers {
+    github = {
+      source  = "integrations/github"
+      version = "4.18.0"
+    }
+  }
+}
+
+resource "github_repository" "this" {
+  name        = var.repository_name
+  description = var.repository_description
+
+  allow_auto_merge       = true
+  allow_merge_commit     = true
+  allow_rebase_merge     = false
+  allow_squash_merge     = false
```

Comment:
We use squash merge allot, so I think we need to set this to true. Or am I thinking wrong?

Reply:
See this conversation, I think? https://github.com/XenitAB/terraform-modules/pull/464#discussion_r760002704